### ISSUE TITLE
Support multi-line values in INI files to enable table parsing

### ISSUE THREAD

**Issue by arthurvd** [Deltares/HYDROLIB-core#143]
This issue is a followup issue after #120.
Example roughness.ini input:
```
# 2021-12-01: Adapted from D-Flow FM UM, C.15, example 1.

[General]
    fileVersion           = 3.01
    fileType              = roughness

[Global]
    frictionId            = Main
    frictionType          = Chezy
    frictionValue         = 45.000

[Branch]
    branchId              = Channel0
    frictionType          = Manning
    functionType          = constant
    frictionValues        = 0.10000

[Branch]
    branchId              = Channel1
    frictionType          = Manning
    functionType          = constant
    numLocations          = 2                     # at two locations
    chainage              = 0.000 100.000
    frictionValues        = 0.20000 0.30000

[Branch]
    branchId              = Channel7
    frictionType          = Chezy
    functionType          = absDischarge
    numLevels             = 3                     # for three values
    levels                = 100.000 200.000 300.000
    numLocations          = 2                     # at two locations
    chainage              = 0.000 300.000
    frictionValues        = 45.00000 55.00000     # values for |Q|=100
                            41.00000 52.00000     # values for |Q|=200
                            40.00000 50.00000     # values for |Q|=300

```
The ```frictionValues``` currently parse correctly for:
- Channel0 -> [0.1]
- Channel1 -> [0.2, 0.4]
But not for:
- Channel7 -> [45, 55]

This prohibits turning the ```FrictBranch.frictionvalues``` from the current ```List[float]``` into a ```List[List[float]]``` (or some other table-like data structure)
Expected result:
- Channel7 -> [[45, 55], [41, 52], [40, 50]]

Request: can we support in ```IniBasedModel``` the parsing of multiline values?
Note: a combination of multiline values (as in this roughness file) and datablocks (as known from .bc files) is currently not necessary.

**Comment by priscavdsluis** [comment]
@arthurvd Can this issue be closed? 
The discussion that was held about this topic came to the conclusion that HYDROLIB-core will **not** support multiline values.

**Comment by priscavdsluis** [comment]
Closing issue (seems like a stale issue); if a user asks for it we can reconsider.
